src/index.html

```diff
@@ -1,17 +1,192 @@
+          Name:
+          <input
+            type="text"
+            name="name-field"
+            id="name-field"
+          >
+        </label>
+
+        <label for="age" class="field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="0"
+            max="0"
+            id="age"
+            required
+          >
+        </label>
+
+        <label for="date-of-birth" class="field">
+          Full date of birth:
+          <input
+            type="text"
+            name="date-of-birth"
+            id="date-of-birth"
+            placeholder="mm / dd / yyyy"
+            required
+          >
+        </label>
+
+        <label for="agreement" class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            placeholder="mm / dd / yyyy"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="email-field" class="field">
+          E-mail:
+          <input
+            type="email"
+            name="email-field"
+            id="email-field"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="password" class="field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about You!</legend>
+        <div for="" class="field">
+          Do you love
+          <label for="y">
+            <input
+              type="radio"
+              name="y"
+              id="y"
+              value="y"
+            >
+            Yes
+          </label>
+          <label for="n">
+            <input
+              type="radio"
+              name="n"
+              id="n"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+
+        <label for="color" class="field">
+          What is your favourite color?
+          <input
+            type="color"
+            name="fav-color"
+            id="color"
+          >
+        </label>
+        <label
+          for="bed"
+          class="field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time-to-bed"
+            id="bed"
+          >
+        </label>
+        <label for="cars" class="field">
+          What are your favourite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
+          </select>
+        </label>
+        <label for="rate" class="form-field">
```

Comment:
```suggestion
        <label for="rate">
```
You are using class `form-field` butyou have no class `form-field` in your css.